src/krc_pb_client.erl

```diff
@@ -41,6 +42,14 @@
 -include_lib("riakc/include/riakc.hrl").
 
 %%%_* Code =============================================================
+delete(Pid, Obj, Options, Timeout) ->
+  case
```

Comment:
This case statement seems useless

Reply:
True, but here I'm just following the convention set by the other functions. There is also one more equally useless case. Although one could argue that this case actually enforces the return value to be either of these two options. I think I'll leave it like this if you're okay with it.